doc/Makefile

```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    = -a -E
```

Comment:
hmm whats the reason for removal?

Reply:
I added these previously to alleviate some build issues I was having. They force a complete rebuild, which as a general rule is nice, but it takes more time. I can remove this change.